src/data/colleges/en.ts

```diff
@@ -111,37 +129,51 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
       'https://www.engineering.cornell.edu/students/undergraduate-students/advising/liberal-studies',
     checker: [
       (course: Course): boolean =>
-        engineeringLiberalArtsDistributions.some(
-          distribution => hasCategory(course, distribution) ?? false
+        Object.values(engineeringLiberalArtsGroups).some(group =>
```

Comment:
Heads up, would be more efficient to do this with a set or variable that's defined beforehand.